src/components/ArchitectHeader.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class ArchitectHeader extends React.Component {
+    render() {
+        return(
+            <p align="center">
+                <a href="//architect.io" target="blank">
+                    <img 
+                        src="https://docs.architect.io/img/logo.svg" 
```

Comment:
could you use the logo from our cdn?
https://cdn.architect.io/logo/horizontal.png